File: PooCC01CNetz/Semana04/Python/Ejercicios02/PasaArregloPrimitivo.py
class PasaArregloPrimitivo:
    def main(self):
        pa = PasaArregloPrimitivo()
        # Solicitar datos
        valores = [4, 5, 1, 3, 2]
        # Logica de mi negocio
        med = pa.media(valores)
        # Obtener resultados
        print("La media de los valores es: " + str(med))

    # Este metodo calcula la media de un conjunto de valores
	# contenidos en un arreglo
    def media(self, vec):
        acum = 0
        for i in range(len(vec)):
            acum += vec[i]
        return acum / len(vec)
    
pasa_arreglo = PasaArregloPrimitivo()
pasa_arreglo.main()

File: PooCC01CNetz/Semana04/Java/Ejercicios02/PasaArregloPrimitivo.java
public class PasaArregloPrimitivo {
	public static void main(String []args) {
		PasaArregloPrimitivo pa = new PasaArregloPrimitivo();
		//Solicitar datos
		double []valores = {4, 5, 1, 3, 2};
		//Logica de mi negocio
		double med = pa.media(valores);
		//Obtener resultados
		System.out.println("La media de los valores es: " + med);
	}
	
	//Este metodo calcula la media de un conjunto de valores
	//contenidos en un arreglo
	public double media(double []vec) {
		double acum = 0;
		for(int i = 0; i < vec.length; i++) {
			acum += vec[i];
		}
		return (acum / vec.length);
	}
}